.travis.yml

```
@@ -0,0 +1,16 @@
+language: ruby
+cache:
+  bundler: true
+  directories:
+    - node_modules
+  yarn: true
+install:
+  - bundle install
+  - nvm install 8.11.2
+  - node -v
+  - npm i -g yarn
+  - yarn
+script:
+  - bundle exec rails assets:precompile
+  - bundle exec rails rspec
```

```suggestion
  - bundle exec rspec
```
I think this line should be as above, I believe the build is failing as a result of this?